app/views/kaminari/_gap.html.haml

```diff
@@ -0,0 +1,2 @@
+%li.page-item.disabled
+  = link_to raw(t 'views.pagination.truncate'), '#', class: 'page-link'
```

Comment:
Do we actually need to customize kaminari view template?

Reply:
Yes to make it fits on mobile view.